src/pages/preferences/preferencesTabs/notificationTab/UserSubscriptions.tsx

```diff
@@ -0,0 +1,151 @@
+import { useMemo, useRef } from "react";
+import { useQuery } from "@apollo/client";
+import styled from "@emotion/styled";
+import {
+  Cell,
+  ExpandedContent,
+  flexRender,
+  HeaderCell,
+  HeaderGroup,
+  HeaderRow,
+  LeafyGreenTableRow,
+  Row,
+  Table,
+  TableBody,
+  TableHead,
+  useLeafyGreenTable,
+} from "@leafygreen-ui/table/new";
+import { SettingsCard, SettingsCardTitle } from "components/SettingsCard";
+import { StyledRouterLink } from "components/styles";
+import { getSubscriberText } from "constants/subscription";
+import {
+  GeneralSubscription,
+  Selector,
+  UserSubscriptionsQuery,
+  UserSubscriptionsQueryVariables,
+} from "gql/generated/types";
+import { USER_SUBSCRIPTIONS } from "gql/queries";
+import { notificationMethodToCopy } from "types/subscription";
+import { resourceTypeToCopy, triggerToCopy } from "types/triggers";
+import { getResourceRoute, processSubscriptionData } from "./utils";
+
+export const UserSubscriptions: React.VFC<{}> = () => {
+  const { data } = useQuery<
+    UserSubscriptionsQuery,
+    UserSubscriptionsQueryVariables
+  >(USER_SUBSCRIPTIONS);
+
+  const subscriptions = useMemo(
+    () => processSubscriptionData(data?.user?.subscriptions),
+    [data]
+  );
+
+  const tableContainerRef = useRef<HTMLDivElement>(null);
+  const table = useLeafyGreenTable<GeneralSubscription | any>({
+    columns,
+    containerRef: tableContainerRef,
+    data: subscriptions ?? [],
+  });
+
+  if (!subscriptions) return null;
```

Comment:
Good point, I added an empty state for when no subscriptions are found. I added disabling the "clear subscriptions" button to EVG-19904 since that button will be moved in that ticket making it easier to conditionally disable.
<img width="1095" alt="image" src="https://github.com/evergreen-ci/spruce/assets/9298431/e1e47fd4-6513-4539-88bd-30e077371fc0">